tensorflow/core/grappler/optimizers/layout_optimizer.cc

```diff
@@ -77,7 +77,10 @@ std::set<string> GetOpsFormatSupported() {
       "MaxPoolGradGradV2",
       "SpaceToDepth",
       "DepthToSpace",
-      "_ROCmFusedConvolutionBiasActivation"};
+      "_ROCmFusedConvolutionBiasActivation",
+      "_ROCmFusedBatchNormActivationInference",
```

Comment:
Is it possible to separate logic related to BN+A kernel fusion into a separate commit?

Reply:
sure...I can break this into two commits